cerbero/build/build.py

```diff
@@ -281,6 +282,9 @@ def configure(self):
             cache = os.path.join(self.config.prefix, '.configure.cache')
             self.config_sh += ' --cache-file=%s' % cache
 
+        if self.enable_gtkdoc:
```

Comment:
Instead of enable_gtkdoc we better do:
```
if self.config.variants.doc:
    if self.use_gtkdoc:
        self.configure_tpl += ' --enable-gtk-doc'
else:
    if self.use_gtkdoc:
        self.configure_tpl += ' --disable-gtk-doc'
endif
```